addons/hr_attendance/models/res_users.py

```diff
@@ -22,5 +23,46 @@ def SELF_READABLE_FIELDS(self):
             'attendance_state',
             'last_check_in',
             'last_check_out',
-            'total_overtime'
+            'total_overtime',
+            'attendance_manager_id',
+            'display_extra_hours',
         ]
+
+    def _clean_attendance_officers(self):
```

Comment:
For performances purposes, it's better to write on the group instead of the user (much less overrides). Untested, but this code should do the same thing.
```py
    def _clean_attendance_officers(self):
        attendance_officers = self.env['hr.employee'].search([('attendance_manager_id', 'in', self.ids)]).attendance_manager_id
        officers_to_remove_ids = self - attendance_officers
        if officers_to_remove_ids:
            self.env.ref('hr_attendance.group_hr_attendance_officer').users = [(3, user.id) for user in officers_to_remove_ids]
```